### PR TITLE
Add TWIR Week 44

### PR DESCRIPTION
**Pull request by cokiengchiara**
- [x] Waiting for EFK @piontec 
- [x] @glitchcrab review cert manager
- [x] @ced0ps review Kong
- [x] @giantswarm/sig-ux please check and add if there are any significant customer-facing UX changes

new addition: Docs
Changes in the last 14 days https://github.com/giantswarm/docs/commits/master
**Respective people who committed, can you (1) check if the changes are noteworthy to announce to customers and (2) add to the announcement file?**
- [x] @whites11 master-az https://github.com/giantswarm/docs/commit/26d7640a3dda95a78e1cec23018f54561dac8c56
- [x] @marians add kgs reference link (maybe use it as an opportunity to promote kgs?) https://github.com/giantswarm/docs/commit/26cd7794c18caad7fb8afa131e6931f2da13666d
- [x] @whites11 remove mentions of ssh for kgs https://github.com/giantswarm/docs/commit/bb4a79f2eb281914b3ced7751508b51f04b2d600 - if important to customers, why?
- [x] @marians add user questions to front matter https://github.com/giantswarm/docs/commit/f3d914226321f6b847880b4cf49c0c31f2b39fa1 
- [x] @corest update apiextensions tag https://github.com/giantswarm/docs/commit/49203b26993c8758d04366eeebaa47ae2ba6b0b4 - not sure if important for customers to know?
- [x] @stone-z , @marians add details on security https://github.com/giantswarm/docs/commit/3e9961bd30376929ef9ea9730ebfcf00ed954c62 - seems relevant?

### CODE REVIEW

File: announcements/2020-Week-44/announcement.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# :zap: This Week in Provider-Independent Announcements (October 16, 2020):zap:
+
+## Giant Swarm Catalog (Managed Apps)
+
+1. [Cert Manager v 2.3.1](https://github.com/giantswarm/cert-manager-app/blob/master/CHANGELOG.md#231---2020-10-29) removes deprecated APIs and upgrades to upstream v1.0.3.
+2. [Kong v0.9.2](https://github.com/giantswarm/kong-app/blob/release-v0.9.x/CHANGELOG.md#092---2020-10-28) adds minReadySeconds to deployment template as an optional parameter 
+
```
Comment (cokiengchiara) [inline announcements/2020-Week-44/announcement.md:7]: Waiting for @piontec EFK release https://github.com/giantswarm/efk-stack-app/blob/master/CHANGELOG.md#unreleased 

Some suggested text

improves reliability of upgrades. It enables PodDisruptionBudget by default, adds tests to ensure ..., and .

@giantswarm/app-squad-efk please help fill in :P

File: announcements/2020-Week-44/announcement.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# :zap: This Week in Provider-Independent Announcements (October 16, 2020):zap:
+
+## Giant Swarm Catalog (Managed Apps)
+
+1. [Cert Manager v 2.3.1](https://github.com/giantswarm/cert-manager-app/blob/master/CHANGELOG.md#231---2020-10-29) removes deprecated APIs and upgrades to upstream v1.0.3.
+2. [Kong v0.9.2](https://github.com/giantswarm/kong-app/blob/release-v0.9.x/CHANGELOG.md#092---2020-10-28) adds minReadySeconds to deployment template as an optional parameter
```
Comment (cokiengchiara) [inline announcements/2020-Week-44/announcement.md:6]:
```suggestion
2. [Kong v0.9.2](https://github.com/giantswarm/kong-app/blob/release-v0.9.x/CHANGELOG.md#092---2020-10-28) adds minReadySeconds (optional parameter) to the deployment template. This helps avoid downtime during rolling updates.
```

**Comment by glitchcrab** [comment]
cert-manager looks good to me!

File: announcements/2020-Week-44/announcement.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# :zap: This Week in Provider-Independent Announcements (October 16, 2020):zap:
+
+## Giant Swarm Catalog (Managed Apps)
+
+1. [Cert Manager v 2.3.1](https://github.com/giantswarm/cert-manager-app/blob/master/CHANGELOG.md#231---2020-10-29) removes deprecated APIs and upgrades to upstream v1.0.3.
+2. [Kong v0.9.2](https://github.com/giantswarm/kong-app/blob/release-v0.9.x/CHANGELOG.md#092---2020-10-28) adds minReadySeconds (optional parameter) to the deployment template. This helps avoid downtime during rolling updates.
+
```
Comment (cokiengchiara) [inline announcements/2020-Week-44/announcement.md:7]:
```suggestion
3. [EFK v0.3.4](https://github.com/giantswarm/efk-stack-app/blob/master/CHANGELOG.md#034---2020-10-30) improves reliability of upgrades. It enables PodDisruptionBudget and builds are now automatically tested with functional testing.
```

**Comment by cokiengchiara** [comment]
@whites11 and @corest confirmed via dm their docs changes not worth mentioning.

File: announcements/2020-Week-44/announcement.md (hunk 3 above)
Comment (piontec) [reply announcements/2020-Week-44/announcement.md:7]: It seems I can only comment a suggestion, not edit it, so:
- I would add info that pod anti-affinity is now enabled by default
- I would add: Please read <https://github.com/giantswarm/efk-stack-app/tree/v0.3.4#important-notes---read-before-you-deploy>

Comment (cokiengchiara) [reply announcements/2020-Week-44/announcement.md:7]: 3. [EFK v0.3.4](https://github.com/giantswarm/efk-stack-app/blob/master/CHANGELOG.md#034---2020-10-30) improves reliability of upgrades. Builds now automatically do functional testing. It also enables pod anti-affinity and PodDisruptionBudget by default. Please [read before you deploy](https://github.com/giantswarm/efk-stack-app/tree/v0.3.4#important-notes---read-before-you-deploy).

**Review by marians — APPROVED** [review]

File: announcements/2020-Week-44/announcement.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# :zap: This Week in Provider-Independent Announcements (October 16, 2020):zap:
```
Comment (marians) [inline announcements/2020-Week-44/announcement.md]: Good catch :)

**Pull request merged**